.github/ISSUE_TEMPLATE/SUPPORT.md

```diff
@@ -0,0 +1,9 @@
+---
+name: 🆘 Support, Help, and Advice
+about: 👉🏽 Need help or tech support? Please don't open an issue! Head tohttps://gitter.im/rollup/rollup or https://stackoverflow.com/questions/tagged/rollupjs.
+
+---
+
+Hey there! If you need help or tech support then this is not the place to
+ask. Please head to [the Rollup Gitter](https://gitter.im/rollup/rollup)
+instead or post a question to https://stackoverflow.com/questions/tagged/rollupjs.
```

Comment:
Maybe we could add something like: If you think Rollup's documentation is unclear, insufficient or wrong, consider creating an issue for the documentation instead.

Reply:
Done, nearly verbatim.

Reply:
👍